internal/controller/cloudsyncmgr/cloudsync.go

```diff
@@ -0,0 +1,36 @@
+package cloudsyncmgr
+
+import (
+	mqttmgr "github.com/lf-edge/edge-home-orchestration-go/internal/common/mqtt"
+)
+
+const (
+	logPrefix = "[cloudsyncmgr]"
+)
+
+// CloudSync is the interface for starting Cloud synchronization
+type CloudSync interface {
+	StartCloudSync(host string) error
+}
+
+//CloudSyncImpl struct
+type CloudSyncImpl struct{}
+
+var (
+	cloudsyncIns *CloudSyncImpl
+)
+
+func init() {
+	cloudsyncIns = &CloudSyncImpl{}
+}
+
+// GetInstance returns cloudSync instaance
+func GetInstance() CloudSync {
+	return cloudsyncIns
+}
+
+// StartCloudSync starts a server in terms of CloudSync
+func (c *CloudSyncImpl) StartCloudSync(host string) (err error) {
+	mqttmgr.StartMQTTClient(host)
```

Comment:
I'm worried about this should be called as go routine. Please see if this line has an issue similar to #160.

Reply:
Done!